components/meter/meter-circle.js

```diff
@@ -57,26 +72,32 @@ class MeterCircle extends MeterMixin(RtlMixin(LitElement)) {
 
 		const primary = this._primary(this.value, this.max) || '';
 		const secondary = this._secondary(this.value, this.max, this.text);
+		const secondaryTextElement = (this.text && !this.textHidden) ? html`<div class="d2l-body-small d2l-meter-circle-text-secondary">${secondary}</div>` : nothing;
 		const textClasses = {
 			'd2l-meter-circle-text-ltr': !this.percent,
 			'd2l-body-standard': true,
 			'd2l-meter-circle-text': true
 		};
 
 		return html`
-			<svg viewBox="0 0 48 48" shape-rendering="geometricPrecision" role="img" aria-label="${this._ariaLabel(primary, secondary)}">
-				<circle class="d2l-meter-circle-full-bar" cx="24" cy="24" r="21"></circle>
-				<circle
-					class="d2l-meter-circle-progress-bar"
-					cx="24" cy="24" r="21"
-					stroke-dasharray="${progressFill} ${space}"
-					stroke-dashoffset="${dashOffset}"
-					visibility="${visibility}"></circle>
+			<div
+				class="d2l-meter-circle"
+				aria-label="${this._ariaLabel(primary, secondary)}">
+				<svg viewBox="0 0 48 48" shape-rendering="geometricPrecision" role="img">
```

Comment:
As Dave B suggested in the now-outdated comments, I think this `role="img"` needs to move to the `div` to align with how `meter-circle` is doing it. Otherwise the `aria-label` on the `div` isn't doing anything.